### PR TITLE
Implemented merge transaction

### PR DESCRIPTION
**Pull request by Pongch**
https://github.com/omisego/plasma-cli/issues/39 🎂 

1. Allows for user to merge 4 or less UTXOs into 1..throws an error if more than 4 UTXOs
```
plasma_cli merge --fromutxo=UTXO_1 --fromutxo=UTXO_2 --fromutxo=UTXO_3 --privatekey="private_key" --fromowner="from-owner --fromwatcher="watcher_path""
```
2. updated sign transaction to take in an array of private key to sign with instead of a single key
3. changed `transaction` to `send`
4. more readable submit response

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,16 +47,21 @@ plasma_cli get utxos --watcher=<watcher URL> --address=<public_address>
 plasma_cli get balance --watcher=<watcher URL> --address=<public_address>
 ```
 
-## Send ETH in the OMG Network
+## Sending ETH UTXO
 
-NOTE: This function will either
+This function will either
```
Comment (jbunce) [inline README.md]: Perhaps it's best to include an additional flag here so the user knows exactly which behaviour will occur?

File: parser/parser.go [hunk 2]
```diff
@@ -25,39 +25,50 @@ import (
...
+	deposit         = kingpin.Command("deposit", "Deposit ETH or ERC20 into the Plamsa MoreVP smart contract.")
+	privateKey      = deposit.Flag("privatekey", "Private key of the account used to send funds into Plasma MoreVP").Required().String()
+	client          = deposit.Flag("client", "Address of the Ethereum client. Infura and local node supported https://rinkeby.infura.io/v3/api_key or http://localhost:8545").Required().String()
+	contract        = deposit.Flag("contract", "Address of the Plasma MoreVP smart contract").Required().String()
+	depositOwner    = deposit.Flag("owner", "Owner of the UTXOs").Required().String()
+	depositAmount   = deposit.Flag("amount", "Amount to deposit in wei").Required().Uint64()
+	depositCurrency = deposit.Flag("currency", "Currency of the deposit. Example: ETH").Required().String()
+
+	transaction      = kingpin.Command("send", "Create a transaction on the OmiseGO Plasma MoreVP network")
+	toowner          = transaction.Flag("toowner", "New owner of the UTXO").Required().String()
```
Comment (jbunce) [inline parser/parser.go]: As `transaction` has been renamed to `send`, it would make sense to change to these variables to reflect that.

Comment (Pongch) [reply parser/parser.go]: good point !

File: README.md (hunk 1 above)
Comment (Pongch) [reply README.md]: so, keep the `NOTE:` flag on ?

Comment (jbunce) [reply README.md]: Perhaps some documentation explaining why UTXO splits -> send to self are used? People used to developing in the Ethereum world may not understand why this is required (thinking of devs new to Plasma). If you don't think this is the right place to describe this behaviour maybe we could add a link to some knowledge base?

Comment (Pongch) [reply README.md]: yeah, devs new to Plasma will have a tough time grasping UTXO from the Readme alone.. I will have a dedicated supporting document for it 👌

**Pull request merged**